### ISSUE TITLE
for some reason when i boot up the SRT and i boot up RE2 the health and timer stay at 0 at all times [BUG] 

### ISSUE THREAD

**Issue by CJ46** [Squirrelies/RE2REmakeSRT#23]
**Computer Information:**
 - OS: [e.g. Windows 10 Version 1809 (Start + 'winver' to find out)]
 - Program Version [e.g. 1.0.2]
 - Game Version [e.g. 1.01]

**Describe the bug**

**Steps to Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by Squirrelies** [comment]
Lacking any details in this bug report, there is not much I can do to look into the issue but I'll ask the questions...

Are you using the latest version of the SRT? Are you running the latest version of the game on Steam? Is the game modified, cracked or pirated?

Anyone of these could be the issue. Older versions of the SRT may have had pointers for older versions of the game. As the game gets updated, sometimes the base pointers are in different position in memory, thus requiring an update to the SRT. The latest SRT is designed to run on the latest version of the game from Steam, as-is (unmodified). Cracks and anything that modifies re2.exe can also mess with the pointer positions causing this issue to occur.

**Comment by gandalfthewhite19890404** [comment]
Same, latest steam version along with latest SRT.

**Comment by R4in84** [comment]
Yes the issue is with the latest steam patch that dropped on the 1st of feb

It broke LiveSplits also but they dropped an update already

**Comment by Squirrelies** [comment]
Original poster lacked any details on their question, subsequent comments were for the same issue as the title but due to either a new patch or potentially modified version of the game. Latest steam release works with the latest SRT release on https://www.neonblu.com/SRT/ as of this date.

**Comment by SmashingQuasar** [comment]
I can confirm I have the exact same issue as OP.
I am playing on the latest version of the game, totally untouched. I reinstalled it entirely to make sure all files are exactly vanilla.
I tried running it both in DX11 and DX12 and it did not change anything.
I tried using full screen and borderless full screen. I usually exclusively play using borderless full screen using DX12.
I also tried activating and deactivating the Steam overlay without getting any result.

Even though I doubt it has anything to do with it, I am running the game on an i9 9900k and a GeForce RTX 3080.
Drivers are all up to date.

Again, I am not using any mod alongside this tool.

I also tried running the game, the host and the tool and also just the tool without the host. None worked.

Hope this helps.
